src/containers/EnterpriseApp/index.jsx

```diff
@@ -20,6 +20,7 @@ const mapStateToProps = (state) => {
     enableLmsConfigurationsScreen: state.portalConfiguration.enableLmsConfigurationsScreen,
     enableReportingConfigurationsScreen: state.portalConfiguration.enableReportingConfigScreen,
     enablePortalLearnerCreditManagementScreen: state.portalConfiguration.enablePortalLearnerCreditManagementScreen,
+    enableContentHighlightsScreen: state.portalConfiguration.enableContentHighlightsScreen,
```

Comment:
Already discussed around this prop here: https://github.com/openedx/frontend-app-admin-portal/pull/882/